dashboard/controller/classrooms.js

```diff
@@ -136,8 +139,8 @@ exports.editClassroom = function(req, res) {
 			req.body.name = req.body.name.trim();
 			req.body.students = req.body.students || [];
 			
-			if (isString(req.body.students)) {
-				req.body.students = [req.body.students]
+			if (typeof req.body.students == 'string') {
```

Comment:
Could you please look into the reason why`req.body.students ` is coming out to be a string.

Reply:
It was quite difficult to pinpoint the issue.

I first tested jquery-multiselect, it was storing single element correctly - in an array.

Then I tested the from. The data is submitted by using traditional form data instead of json data submitted by using fetch/request api.

When I used jquery multiselect to select two elements and submit data using form method='POST':
![Screenshot from 2019-03-10 21-49-10](https://user-images.githubusercontent.com/24666770/54088050-8f900a00-437f-11e9-8835-52c36cd793e5.png)

When I selected single element and submitted the form method='POST':
![Screenshot from 2019-03-10 21-49-34](https://user-images.githubusercontent.com/24666770/54088056-a3d40700-437f-11e9-869d-fb5bfe731e0f.png)

As you can see the way form data is submitted, it's not possible for the server parser to identify that element with unique key in the request is an array or not.